### ISSUE TITLE
Privacy feature "Run"

### ISSUE THREAD

**Issue by mphasize** [levanto-financial/ember-oembed#10]
This would fit nicely in the component level as well: A feature that doesn't load the embed immediately, but gives the user the option to "run" an embed via mouse-click or tap.

This could help protect users privacy by default + save a lot of bandwidth in busy websites.

What do you think?
Cheers!

**Comment by mphasize** [comment]
@mike-north Hi Mike, sorry for poking, but would be great to get quick feedback from you before I attempt a pull request on this idea. Thanks!  :)

**Comment by mike-north** [comment]
Yeah this sounds fine. You have the default behavior I would want
